### PR TITLE
CTOR-397 : [Plugin(storage::wd::nas::snmp) - Modes(volumes, listvolumes, hardware) : add support for ex4100 model

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Add support for ex4100 model in Modes: volumes, listvolumes and hardware

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md [hunk 1]
```diff
@@ -5,49 +5,90 @@ title: WD NAS SNMP
...
+
+### Modèles d'équipement supportés
+
+* EX2
+* EX2 Ultra
+* EX 4100
+* PR 2100
+* PR 4100
+
+> D'autres modèles d'équipement peuvent être compatibles mais nous ne pouvons le garantir à priori
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:18]:
```suggestion
> D'autres modèles d'équipement peuvent être compatibles, mais nous ne pouvons le garantir à priori.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md [hunk 2]
```diff
@@ -5,49 +5,90 @@ title: WD NAS SNMP
...
 ### Règles de découverte
 
-| Nom de la règle                    | Description                                     |
-|:-----------------------------------|:------------------------------------------------|
-| HW-Storage-Wd-Nas-SNMP-Volume-Name | Découvre les volumes et supervise l'utilisation |
+#### Découverte d'hôtes
+
+| Nom de la règle | Description                                                                                                                                                                                                                                                 |
+|:----------------|:------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **HW-Storage-Wd-Nas-SNMP-custom** host template |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:58]:
```suggestion
| SNMP Agents     | Découvre les ressources via un scan réseau SNMP. Installez le connecteur [Generic SNMP](./applications-protocol-snmp.md) pour obtenir la règle de découverte et créez un modificateur pour le modèle d'hôte **HW-Storage-Wd-Nas-SNMP-custom** |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md [hunk 3]
```diff
@@ -5,49 +5,90 @@ title: WD NAS SNMP
...
-| *volume_name*#volume.space.free.bytes       | B     |
-| *volume_name*#volume.space.usage.percentage | %     |
+| Métrique                                | Unité |
+|:----------------------------------------|:------|
+| *volumes*#volume.space.usage.bytes      | B     |
+| *volumes*#volume.space.free.bytes       | B     |
+| *volumes*#volume.space.usage.percentage | %     |
 
 </TabItem>
 </Tabs>
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:94]: prérequis standard SNMP

File: pp/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md [hunk 4]
```diff
@@ -5,49 +5,89 @@ title: WD NAS SNMP
...
-| *volume_name*#volume.space.free.bytes       | B     |
-| *volume_name*#volume.space.usage.percentage | %     |
+| Metric name                             | Unit  |
+|:----------------------------------------|:------|
+| *volumes*#volume.space.usage.bytes      | B     |
+| *volumes*#volume.space.free.bytes       | B     |
+| *volumes*#volume.space.usage.percentage | %     |
 
 </TabItem>
 </Tabs>
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:93]: standard SNMP prerequisites

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: pp/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md (hunk 4 above)
Comment (lucie-dubrunfaut) [reply pp/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:93]: image

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md (hunk 3 above)
Comment (lucie-dubrunfaut) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:94]: image

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md [hunk 5]
```diff
@@ -56,20 +97,18 @@ pour en savoir plus sur la découverte automatique de services et sa [planificat
 
 ### Configuration SNMP
 
-Afin de superviser votre **WD NAS** en SNMP,  il est nécessaire de configurer l'agent sur l'équipement.
+Pour utiliser ce connecteur de supervision, le service SNMP doit démarré et configuré sur le serveur **WD NAS**.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:100]:
```suggestion
Pour utiliser ce connecteur de supervision, le service SNMP doit être démarré et configuré sur le serveur **WD NAS**.
```

Comment (cg-tw) [reply i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/hardware-storage-wd-nas-snmp.md:100]: il manque l'histoire du manufacturer/publisher

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Pull request merged**